packages/ActionBar/tests/Actionbar.cypress.js

```diff
@@ -0,0 +1,96 @@
+describe("ActionBar", () => {
+  beforeEach(() => {
+    cy.visitStorybook("action-bar--showcase");
+  });
+
+  it("Should render", () => {
+    cy.get("span")
+      .contains("Filter")
+      .should("be.visible");
+    cy.get("span")
+      .contains("Sort")
+      .should("be.visible");
+    cy.get("span")
+      .contains("Arrange")
+      .should("be.visible");
+  });
+
+  it("Should display the differant filtering options", () => {
+    cy.get("span")
+      .contains("Filter")
+      .click();
+    cy.contains("Add a field to filter by").click();
+    cy.get("select")
+      .eq(0)
+      .select("Goals")
+      .select("Name")
+      .select("Status")
+      .select("Country")
+      .select("Joined by")
+      .select("Shareable")
+      .select("Level");
```

Comment:
just curious why do you want to select each item once

Reply:
Yes, was checking to see if all of the items were there.